app/web/features/profile/view/UserCard.tsx

```diff
@@ -16,19 +18,23 @@ import UserTabContext from "./UserTabContext";
 
 const REQUEST_ID = "request";
 
-export const sectionLabels = (t: TFunction, user?: User.AsObject) => ({
-  about: t("profile:heading.about_me"),
-  home: t("profile:heading.home"),
-  references: (
-    <Box display="flex" alignItems="center" gap={1}>
-      <Typography component="span">
+export const sectionLabels = (
+  t: TFunction,
+  user?: User.AsObject,
+  isSuperuser?: GetAccountInfoRes.AsObject["isSuperuser"],
+) => {
+  return {
+    about: t("profile:heading.about_me"),
+    home: t("profile:heading.home"),
+    references: (
+      <Box display="flex" alignItems="center" gap={1}>
         {t("profile:heading.references")}
-      </Typography>
-      <StyledNumReferences>{user?.numReferences}</StyledNumReferences>
-    </Box>
-  ),
-  mod: t("global:mod"),
-});
+        <StyledNumReferences>{user?.numReferences}</StyledNumReferences>
+      </Box>
+    ),
```

Comment:
Just fixing the References while I'm here as the fontSize was bigger than the others.